config/locales/fr.yml

```diff
@@ -19,6 +19,16 @@ fr:
     report_abuse: Signaler un abus
     search: Recherche
     success_settings: Successfully updated settings.
+    counted_organization:
+      one: "%{count} organisation"
+      other: "%{count} organisations"
+    counted_user:
+      one: "%{count} personne"
+      other: "%{count} personnes"
+    not_following: "Vous ne suivez personne"
+    following_everyone: "Vous les suivez tous"
```

Comment:
Happy to defer to you here, but to clarify a bit: the context here _is_ closer to `following them all` (following everyone in a list of suggested folk). I think English "everyone" has enough ambiguity to be applicable in either case, but my French is lousy and context is hard. (And, of course, that context is hard-to-impossible to convey well in the `yaml`. Debatably, since we are still working out our i18n focus, I maybe should not even be using i18n for these bits — these keys will almost certainly need to be re-located — but i18n handles the "person:people::organization:organizations" bits better than our work-around did, so it felt like a good time to make the push.)